### PR TITLE
Add capacity vars

### PR DESCRIPTION
**Pull request by robertpietzcker**

### CODE REVIEW

**Comment by robertpietzcker** [comment]
(I didn't know how to repeat the checks after updating the indentation, so I did a new PR)

File: definitions/variable/technology/technologies.yaml [hunk 1]
```diff
@@ -113,6 +116,9 @@
 - Capacity|Liquids|Oil:
     description: Total installed (available) capacity of oil refining plants
     unit: GW
+- Capacity|Liquids|Hydrogen:
+    description: Total installed (available) capacity of hydrogen to liquids (synguel) plants
```
Comment (danielhuppmann) [inline definitions/variable/technology/technologies.yaml:120]:
```suggestion
    description: Total installed (available) capacity of hydrogen-to-liquids (synfuel) plants
```

File: definitions/variable/technology/technologies.yaml [hunk 2]
```diff
@@ -44,6 +44,9 @@
 - Capacity|Gases|Coal|w/o CCS:
     description: Total installed (available) capacity of coal to gas plants without CCS
     unit: GW
+- Capacity|Gases|Hydrogen:
+    description: Total installed (available) capacity of hydrogen to methane (syngas) plants
```
Comment (danielhuppmann) [inline definitions/variable/technology/technologies.yaml:48]:
```suggestion
    description: Total installed (available) capacity of hydrogen-to-methane (syngas) plants
```

**Review by danielhuppmann — CHANGES_REQUESTED** [review]
One typo, two minor edits in the description to improve readability.

**Comment by robertpietzcker** [comment]
thanks, @danielhuppmann !

**Review by danielhuppmann — APPROVED** [review]
Thanks!

**Pull request merged**